x-pack/plugins/uptime/common/lib/helper/test_helpers.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { Moment } from 'moment-timezone';
+
+export function mockMoment() {
```

Comment:
are we adding these or these moved from someplace else? 
i think we already had these in uptime somewhere.

Reply:
This was not meant to be pushed. Thanks for the catch.